spring-cloud-gcp-bigquery/src/test/java/org/springframework/cloud/gcp/bigquery/integration/outbound/BigQueryFileMessageHandlerIntegrationTests.java

```diff
@@ -145,10 +152,35 @@ public void testLoadFile_cancel() {
 		ListenableFuture<Job> jobFuture =
 				(ListenableFuture<Job>) this.messageHandler.handleRequestMessage(message);
 		jobFuture.cancel(true);
+		assertThat(this.taskScheduler.getScheduledThreadPoolExecutor().getQueue()).hasSize(1);
 
 		await().atMost(10, TimeUnit.SECONDS).untilAsserted(() -> {
 			// This asserts that the BigQuery job polling task is no longer in the scheduler after cancel.
 			assertThat(this.taskScheduler.getScheduledThreadPoolExecutor().getQueue()).hasSize(0);
 		});
 	}
+
+	@Configuration
+	static class BigQueryTestConfiguration {
+
+		@Bean
+		ThreadPoolTaskScheduler taskScheduler() {
```

Comment:
There is a default `taskScheduler` bean created by the Spring Integration infrastructure. You don't need this as long as you use `@EnableIntegration` on the `@Configuration` class